src/focus_manager/txm_platform.js

```diff
@@ -585,7 +585,12 @@ export class TXMPlatform {
 
             // Android in the user agent is true for both Android mobile and AndroidTV
             // Note also that we don't consider FireTV to be Android, to avoid confusing it with mobile devices.
-            if (self.supportsTouch) {
+
+            // The MIBOX is mobile hardware in a settop box! And so we need to hard code our knowledge that is
+            // actually supposed to be an android TV DEVICE.
+            const isMIBox = userAgent.indexOf('MIBOX') >= 0;
```

Comment:
Sort of minor, but suggest checking it starts with MIBOX instead of contains.  
Also consider case-insensitivity.  Searching suggests 3rd generation does return "MIBOX3" too, so this may not be needed.

Reply:
I want case insensitive, matching both MIBOX3 and MIBOX4 is ok
I can look at a regex with \b though